### PR TITLE
Expose endpoint for fetching single desktop

### PR DESCRIPTION
**Pull request by ibeckermayer**
Exposes an endpoint for fetching a single desktop by name. 

Depends on https://github.com/gravitational/webapps/pull/448 (`desktop` was changed to `desktops` in websocket addr).

### CODE REVIEW

File: lib/web/apiserver.go [hunk 1]
```diff
@@ -410,7 +410,8 @@ func NewHandler(cfg Config, opts ...HandlerOption) (*RewritingHandler, error) {
 
 	// Desktop access endpoints.
 	h.GET("/webapi/sites/:site/desktops", h.WithClusterAuth(h.getDesktopsHandle))
-	h.GET("/webapi/sites/:site/desktop/:desktopUUID/connect", h.WithClusterAuth(h.handleDesktopAccessWebsocket))
+	h.GET("/webapi/sites/:site/desktops/:desktopUUID", h.WithClusterAuth(h.getDesktopHandle))
```
Comment (alex-kovoy) [inline lib/web/apiserver.go:413]: `desktopUUID` -> `desktopID`

**Review by alex-kovoy — APPROVED** [review]

File: lib/web/apiserver.go (hunk 1 above)
Comment (zmb3) [reply lib/web/apiserver.go:413]: I actually have a change in progress to rename this to `desktopName`. It's not a UUID.

File: constants.go [hunk 2]
```diff
@@ -739,3 +739,6 @@ const UserSingleUseCertTTL = time.Minute
 // StandardHTTPSPort is the default port used for the https URI scheme,
 // cf. RFC 7230 § 2.7.2.
 const StandardHTTPSPort = 443
+
+// StandardRDPPort is the default port used for rdp.
```
Comment (zmb3) [inline constants.go:743]:
```suggestion
// StandardRDPPort is the default port used for RDP.
```

File: lib/web/servers.go [hunk 3]
```diff
@@ -80,3 +80,20 @@ func (h *Handler) getDesktopsHandle(w http.ResponseWriter, r *http.Request, p ht
 
 	return ui.MakeDesktops(windowsDesktops), nil
 }
+
+// getDesktopsHandle returns a desktop.
```
Comment (zmb3) [inline lib/web/servers.go:84]:
```suggestion
// getDesktopHandle returns a desktop.
```

I would probably name this `handleGetDesktop` instead though.

File: lib/web/ui/server.go [hunk 4]
```diff
@@ -207,28 +210,41 @@ type Desktop struct {
 	Labels []Label `json:"labels"`
 }
 
+// MakeDesktops converts a desktop from its API form to a type the UI can display.
```
Comment (zmb3) [inline lib/web/ui/server.go:213]:
```suggestion
// MakeDesktop converts a desktop from its API form to a type the UI can display.
```

**Review by zmb3 — CHANGES_REQUESTED** [review]
LGTM other than the `desktopUUID` name - let's change that to `desktopName` and make sure you update all references.

File: lib/web/servers.go (hunk 3 above)
Comment (ibeckermayer) [reply lib/web/servers.go:84]: It seems as though we haven't been being particularly disciplined about this, but this aligns with @kimlisa 's suggested naming convention.

https://github.com/gravitational/teleport/blob/master/lib/web/apiserver.go#L393-L409

**Review by zmb3 — APPROVED** [review]

**Pull request merged**